polaris-react/src/components/IndexTable/IndexTable.module.css

```diff
@@ -1026,7 +1022,10 @@
     padding: var(--p-space-150);
 
     &:first-child {
-      padding-left: var(--p-space-300);
+      /* stylelint-disable-next-line -- specificity overrides */
+      padding-left: var(--pc-index-table-checkbox-offset-left);
+      /* stylelint-disable-next-line -- specificity overrides */
+      padding-right: var(--pc-index-table-checkbox-offset-right);
```

Comment:
This is `--p-space-200` (ie; `8px`). Before it was getting `--p-space-150` (`6px`) from above.

The 2px difference was being added to the inner width of the element, making it `20px` wide even though the checkbox is only `18px` wide.

Later, when the checkbox is rendered, it was then being wrapped in a `flex` container whos only job was to re-center the checkbox due to the added `2px`.

So, by setting the correct padding here, we can remove the unnecessary div later 👍